### ISSUE TITLE
vcf2maf package (v1.6.21) is broken an does not correspond to the Github version with the same version number.

### ISSUE THREAD

**Issue by stefmldk** [bioconda/bioconda-recipes#39598]
The vcf2maf package (main is v1.6.21) is broken. Even though the version is the same as the latest version in GitHub, the code in vcf2maf.pl from Bioconda is NOT the same as vcf2maf.pl from GitHub and it fails right out of the box. I have only checked vcf2maf.pl.

vcf2maf.pl is a wrapper for Ensemble's VEP and operates by constructing commands for VEP. In the vcf2maf.pl script hosted by Bioconda, a default command string is constructed that contains a flag that is understood by vcf2maf.pl but not VEP, which therefore fails.

Conversely, the vcf2maf.pl from the vcf2maf GitHub works as expected. This has clearly been missed in the review process - please correct this as soon as possible. Thanks a lot, guys.

**Comment by wm75** [comment]
Would you mind providing a bit more details:
- What's the problematic flag?
- Is this happening with *any* input or just in specific circumstances?
- When claiming that there a re code differences between github and bioconda, are you referring to the same release (1.6.21) of the code?

**Comment by stefmldk** [comment]
Hi wm75
I don't know exactly what is going on here, because I don't see a difference any longer between the Bioconda version and the one from GitHub. When I submitted the issue, the Bioconda version of vcf2maf.pl appeared edited in several ways (I looked at vcf2maf.pl in the environment where the vcf2maf.pl call failed):

First of all, when the default VEP command was constructed (line 462), a command line option that only vcf2maf understands (not VEP) was added to the VEP command and this of course caused the VEP command to fail (the thrown error matched this option – but unfortunately I did not take note of it). Secondly, the help text (the one written when calling vcf2maf.pl with the --help option) was formatted differently.

It is possible that the vcf2maf in my environment had been installed from some other repo than Bioconda – although I believe I have been explicitly requesting it from Bioconda. Another possibility is that at some point between submitting the issue and now, I transferred here from a parallel universe. I lean to the latter, so let me just get the formalities in order: Greetings fellow earthlings! I come in peace!
